test/snapshot.ts

```diff
@@ -233,14 +233,5 @@ describe('Snapshot', () => {
 
       snapshot.delete(done);
     });
-
```

Comment:
So...  what's up with the deleted test?

Reply:
They were useless. We would do a `callback = callback || noop` but callback will never be undefined because omitting it just users the promisified version.